certs_governance/src/metadata.rs

```diff
@@ -0,0 +1,75 @@
+//! Module MetaData
+//!
+//! Module for obtaining and modifying the metadata fields.
+use crate::storage_types::{CertData, DataKey, Opt, Status};
+use soroban_sdk::{map, Address, Bytes, Env, Map, Vec};
+use uuid::Uuid;
+
+pub fn read_file_storage(e: &Env) -> Bytes {
+    let key = DataKey::FStorage;
+    e.storage().get_unchecked(&key).unwrap()
+}
+
+pub fn write_file_storage(e: &Env, d: Bytes) {
+    let key = DataKey::FStorage;
+    e.storage().set(&key, &d);
+}
+
+pub fn read_name(e: &Env) -> Bytes {
+    let key = DataKey::Name;
+    e.storage().get_unchecked(&key).unwrap()
+}
+
+pub fn write_name(e: &Env, d: Bytes) {
+    let key = DataKey::Name;
+    e.storage().set(&key, &d)
+}
+
+pub fn read_revocable(e: &Env) -> bool {
+    let key = DataKey::Revocable;
+    e.storage().get_unchecked(&key).unwrap()
+}
+
+pub fn write_revocable(e: &Env, d: bool) {
+    let key = DataKey::Revocable;
+    e.storage().set(&key, &d)
+}
+
+pub fn read_expiration_time(e: &Env) -> Option<u64> {
+    let key = DataKey::ExpTime;
+    e.storage().get_unchecked(&key).unwrap()
+}
+
+pub fn write_expiration_time(e: &Env, d: Option<u64>) {
+    let key = DataKey::ExpTime;
+    e.storage().set(&key, &d)
+}
+
+pub fn create_receivers(e: &Env, d: Vec<Address>) {
```

Comment:
What do you think about using a more descriptive name for the `d` argument?

Reply:
Yes, we agree

Reply:
I agree